packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -365,38 +365,46 @@ export default {
 
                 await this.handleUpdateCheckout();
 
-                if (this.isFulfillable) {
-                    await this.submitOrder();
-
-                    this.$emit(EventNames.CheckoutSuccess, eventData);
-
-                    this.$logger.logInfo(
-                        'Consumer Checkout Successful',
-                        this.$store,
-                        eventData
-                    );
-
-                    this.redirectToPayment();
-                } else {
-                    this.$logger.logWarn(
-                        'Consumer Checkout Not Fulfillable',
-                        this.$store,
-                        eventData
-                    );
-                }
+                await this.handleFulfillableContext(eventData);
             } catch (thrownErrors) {
                 eventData.errors = thrownErrors;
 
                 this.$emit(EventNames.CheckoutFailure, eventData);
 
-                this.$logger.logError(
-                    'Consumer Checkout Failure',
-                    this.$store,
-                    eventData
-                );
+                this.logInfo('Consumer Checkout Failure', eventData);
             }
         },
 
+        async handleFulfillableContext (eventData) {
+            if (this.isFulfillable) {
+                await this.processOrderIsFulfillable(eventData);
+            } else {
+                this.processOrderNotFulfillable(eventData);
+            }
+        },
+
+        processOrderNotFulfillable (eventData) {
+            this.logInfo('Consumer Checkout Not Fulfillable', eventData);
+        },
+
+        async processOrderIsFulfillable (eventData) {
+            await this.submitOrder();
+
+            this.$emit(EventNames.CheckoutSuccess, eventData);
+
+            this.logInfo('Consumer Checkout Successful', eventData);
+
+            this.redirectToPayment();
+        },
+
+        logInfo (message, eventData) {
```

Comment:
Are we sure we want to reduce the three current log levels we're currently producing out of this component to 'info' level only?

Reply:
My mistake. @FayeCarter pointed it out earlier to me. Just fixing it now.